.github/workflows/azure-fd-blob-fn-app.yml

```diff
@@ -13,7 +13,7 @@ on:
 env:
   NODE_VERSION: "12.x" # set this to the node version to use (supports 8.x, 10.x, 12.x)
   AZURE_FUNCTIONAPP_PACKAGE_PATH: "./api" # set this to the path to your web app project, defaults to the repository root
-  AZURE_REACT_PACKAGE_PATH: "./src" # set this to the path to your web app project, defaults to the repository root
+  AZURE_REACT_PACKAGE_PATH: "./client/src" # set this to the path to your web app project, defaults to the repository root
```

Comment:
iIt might be better if we drop the /src here, append /src on line 54, and use the  AZURE_REACT_PACKAGE_PATH variable on line 127. that way we only have the path defined in one place. What do you think?

Reply:
I'm trying something. I don't think we need the src path here at all since client's npm commands live at the `client` directory level anyways. Node/npm is just smart enough to let you run the various npm commands from anywhere within the npm project dir (so since src is a child of client its valid)

Reply:
Hmm seems to have broken. Maybe I was mistaken

Reply:
Could you make sense of this? Maybe I'm missing something here. I don't know why client `src/` path is dependent on the deploy step.